packages/paste-core/components/button/src/types.ts

```diff
@@ -34,30 +35,70 @@ export interface ButtonContentsProps {
 }
 
 export interface DirectButtonProps extends HTMLPasteProps<'button'>, Pick<BoxProps, 'element'> {
+  /**
+   * The HTML tag to replace the default `<button>` tag.
+   * @default button
+   */
   as?: keyof JSX.IntrinsicElements;
   buttonState: ButtonStates;
   children: React.ReactNode;
+  /**
+   * Prevent actions from firing on this Button
+   * @default false
+   */
   disabled?: boolean;
+  /**
+   * Sets the Button width to 100% of the parent container.
+   * @default false
+   */
   fullWidth?: boolean;
+  /**
+   * A URL to route to. Must use as="a" for this prop to work.
+   * @default null
+   */
   href?: string;
+  /**
+   * Sets the aria-pressed attribute. Must be used with 'secondary' or 'secondary_icon' variants.
+   * @default false
+   */
   pressed?: boolean;
+  /**
+   * @default default
+   */
   size: ButtonSizes;
   tabIndex?: ButtonTabIndexes;
   target?: string;
+  /**
+   * If the Button is inside a <form>: use 'submit'. Otherwise use 'button' (default).
```

Comment:
suggestion: clarify comment to say only use submit button once in a form

Reply:
also this form is literally a form

<img width="614" alt="Screenshot 2023-09-15 at 5 26 47 PM" src="https://github.com/twilio-labs/paste/assets/75342690/faefd5b2-65b1-41bc-aee4-6db7533c4504">